<launch>
    <node name="ax2550_joy_tester" pkg="ax2550_python" type="ax2550_joy_tester.py" output="screen">
      <param name="button_as_toggle" value="3"/>
    </node>
    <node name="ax2550_teleop" pkg="ax2550_python" type="ax2550_teleop.py" output="screen" />
    <node name="joy_node" pkg="joy" type="joy_node" output="screen">
	<param name="dev" value="/dev/joystick"/>
    </node>
</launch>
